.github/workflows/companion-deploy.yml

```diff
@@ -23,6 +23,11 @@ jobs:
           tag-sha: true
           tag-match: |
             \d{1,3}.\d{1,3}.\d{1,3}
+      - name: Get Companion version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@main
```

Comment:
is `jq` pre-installed on GHA?

Reply:
It is yes, but it can easily be replaced with `node` if you prefer (`node -p '"::set-output name=current_version::" + require("./packages/@uppy/companion/package.json")'`). However, using `GITHUB_REF` is arguably even simpler.

Reply:
`GITHUB_REF` indeed seems to be perfect.